docs/source/tutorial.rst

```diff
@@ -63,6 +63,19 @@ Or choose a model from a pre-loaded library, like ``Modelica Standard Library``:
 
 This will return an instance of a ``Model`` class object.
 
+Setting logging levels for analysis
+-----------------------------------
+
+The amount of logging that should be output by the client can be set by specifying the log level in the logger. 
+The log is printed to the standard output, normally the terminal window from which the compiler is invoked.
+The available log levels are 'INFO'(default), 'ERROR', 'WARNING', 'CRITICAL' and 'DEBUG'. The below code snippet could be used to
+alter log levels::
+   
+   from modelon.impact.client import logger
```

Comment:
You typically do it like this:
```
import logging

logging.getLogger('modelon.impact.client').setLevel(logging.ERROR)
```

This is much more flexable and can always be done on a Python module, regardless of what it contains with regards to logging.